### DISCUSSION TITLE
How to change the preview text panel colorscheme

### DISCUSSION THREAD

**Discussion by harisonkhlil** [sxyazi/yazi#818] [1. Q&A] [answered]
<img width="1308" alt="image" src="https://github.com/sxyazi/yazi/assets/70578017/ff668805-07de-4eaf-880b-0bc97ec149d3">

The overall color scheme of the scheme is tokyonight Why is the preview text window not in this color scheme?

How to change the preview text panel colorscheme

**Comment by lpnh** [comment]
Hi! Could this be related to the following [issue](https://github.com/sxyazi/yazi/issues/557) by any chance?

**Reply by harisonkhlil** [reply]
It's not this one you're talking about but thanks anyway

**Comment by harisonkhlil** [comment, marked as answer]
The color scheme found [here](https://github.com/folke/tokyonight.nvim/tree/main/extras/sublime) and then adding `syntect_theme = "tokyonight_night.tmTheme"` to do it in `theme.toml`!
